conda/recipes/rapids/meta.yaml

```diff
@@ -29,12 +29,6 @@ requirements:
     - cuda-version ={{ cuda_version }}
   run:
     - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
-    {% if cuda_major == "11" %}
-    - cuda-python {{ cuda11_cuda_python_version }}
-    - cudatoolkit
```

Comment:
Just checking... was removing `cudatoolkit` for CUDA 11.x packages also intentional here?

Reply:
I saw this in the description:

> I also removed cudatoolkit because that is a transitive dependency of RAPIDS libraries, and only needed for CUDA 11

But this is in a block that's CUDA-11-specific.

Reply:
Yes, it was intentional. We can revert that change if we want to keep this PR narrower, but it falls into the same bucket of "don't put packages in the `rapids` metapackage if we expect it to be supplied by RAPIDS libraries, because it has the potential to make the solver do bad things with its priorities."

There are other packages that also fall in this category, like ptxcompiler (which is a hard dependency of cudf's CUDA 11 package).

The `rapids` metapackage (imo) should be _only_ RAPIDS libraries, plus any other constraints needed to make the environment simpler for conda/mamba to resolve or more reproducible. If others agree, we could remove things like `numba`, `numpy`, etc. and rely on RAPIDS libraries to pin their own compatible versions of those. We certainly don't want the `rapids` metapackage to make the pinnings for packages like those any tighter than what RAPIDS libraries collectively support.

Reply:
Ok thank you for that explanation! Just wanted to double-check that this was intentional, and that I understood it fully.